src/components/DataController/DataController.js

```diff
@@ -203,9 +201,11 @@ var DataController = React.createClass({
             <TimeOfYearSelector onChange={this.updateDataTableTimeOfYear} />
           </Col>
         </Row>
-        <DataTable data={statsData} />
+          <DataTable data={statsData} />
+          <Button onClick={exportTableDataToWorksheet.bind(this, statsData, 'xlsx')}>Export To XLSX</Button>
```

Comment:
These buttons might look better on the UI with a bit of padding between them and the DataTable, and being horizontally centered.